AUTHORS.rst

```diff
@@ -0,0 +1,14 @@
+=======
+Credits
+=======
+
+Development Lead
+----------------
+
+* Mr Gambal <mr_gambal@outlook.com>
+* Dmitry Chaplinsky <chaplinsky.dmitry@gmail.com>
+
+Contributors
+------------
+
+* Volodymyr Hotsyk <gotsyk@gmail.com>
```

Comment:
Don't hesitate to add yourself to authors list. We're completely equal in that good intention.